Sources/MapboxNavigation/RouteVoiceController.swift

```diff
@@ -107,23 +111,6 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         }
     }
     
-    func safeMixAudio(instruction: SpokenInstruction?, failure: AudioControlFailureHandler) {
-        do {
-            try tryMixAudio()
-        } catch {
-            let wrapped = SpeechError.unableToControlAudio(instruction: instruction, action: .mix, underlying: error)
-            failure(wrapped)
-            return
-        }
-    }
-    
-    func tryMixAudio() throws {
-        guard speechSynthesizer.managesAudioSession else { return }
```

Comment:
This should be taken into account or new just break apps audio session without anyway to interfere.